dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -350,55 +370,65 @@ URL findClassLocation(String className) throws ClassNotFoundException {
   }
 
   /**
-   * @param jarFilePaths absolute paths to jar files
-   * @return map of jar file paths to classes defined in them
+   * Returns mapping from jar files to the names of their defining classes.
+   *
+   * @param jarPaths absolute paths to jar files
    */
-  private static ImmutableSetMultimap<Path, String> jarFilesToDefinedClasses(
-      List<Path> jarFilePaths) throws IOException {
+  @VisibleForTesting
+  static ImmutableSetMultimap<Path, String> scanJarFilesToDefinedClasses(
```

Comment:
I don't follow this method name. Maybe mapJarToClasses? 

In general try to avoid duplicating the method arguments (here JarFiles/jarPaths) in method names unless it's necessary for clarity or correctness.